app/src/main/java/com/sunasterisk/youtubebackground/screen/fragment/gamingtrending/GamingTrendingFrag.kt

```diff
@@ -0,0 +1,24 @@
+package com.sunasterisk.youtubebackground.screen.fragment.gamingtrending
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+import com.sunasterisk.youtubebackground.R
+
+class GamingTrendingFrag : Fragment() {
```

Comment:
```suggestion
class GamingTrendingFragment : Fragment() {
```
Sửa các fragment khác luôn